src/HeapDump/HeapDump.csproj

```diff
@@ -0,0 +1,83 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- net451 is actually just another net45 build build with the platform set to x64 -->
```

Comment:
This limitation is due to a bug in one or more of dotnet/project-system, dotnet/roslyn, dotnet/sdk, and/or Microsoft/msbuild. It is my understanding that the teams do want to allow configurations on facets other than the predefined set of target frameworks. Ideally, a future update to this project file would have the following:

```xml
<TargetFramework>net45</TargetFramework>
<Platforms>AnyCPU;x64</Platforms>
```

Part of the ongoing effort to understand scenarios for the new project system is implementing things like we have in this file and making sure bugs are filed to make improvements based on real-world use cases. In this case, dotnet/sdk#350 suggests what we want might possible. dotnet/project-system#1938 (and the issues linked to it) track various issues related to multi-building on a facet other than just the target framework.

Given that the "trickery" is limited to a small number of lines, it seems like this approach should be fine even without a follow-up issue. But if it's concerning I could create an issue to make adjustments as soon as the project system supports it.